src/Coordination/KeeperDispatcher.cpp

```diff
@@ -394,8 +394,10 @@ void KeeperDispatcher::setResponse(int64_t session_id, const Coordination::ZooKe
         if (session_id_resp.server_id != server->getServerID() || !new_session_id_response_callback.contains(session_id_resp.internal_id))
             return;
 
-        auto callback = new_session_id_response_callback[session_id_resp.internal_id];
-        callback(response, request);
+        auto callback = new_session_id_response_callback.find(session_id_resp.internal_id);
```

Comment:
we check with contains that the internal id is present so this check is not necessary.